website/docs/internals/json-format.mdx

```diff
@@ -49,7 +49,7 @@ The JSON output format consists of the following objects and sub-objects:
 
 ## State Representation
 
-Because state does not currently have any significant metadata not covered by the common values representation ([described below](#values-representation)), the `<state-representation>` is straightforward:
+Because state does not  have any significant metadata not covered by the common values representation ([described below](#values-representation)), the `<state-representation>` uses the following format:
```

Comment:
```suggestion
Because state does not have any significant metadata not included in the common [values representation](#values-representation), the `<state-representation>` uses the following format:
```